torch/_inductor/codegen/triton.py

```diff
@@ -1673,14 +1721,17 @@ def final_argreduce(buffer, result_var, value, index):
         result_var.mask_vars = {var for var in masks if var[0] != "r"}
         cond = " & ".join(masks)
 
+        def where_cond(tval, fval):
+            if not cond:
```

Comment:
This check doesn't seem to have existed before, so I'm wondering how could the previous version of the code have worked with an empty `cond` inlined in the literal `"tl.where(...)"`.

Reply:
Adding a maximum size for RBLOCK means the mask elimination pass sometimes results in no mask.  Before RBLOCK always existed.